overlord/assertstate/assertstate.go

```diff
@@ -289,6 +289,24 @@ func Publisher(s *state.State, snapID string) (*asserts.Account, error) {
 	return a.(*asserts.Account), nil
 }
 
+// PublisherAccount returns the store account information from the publisher assertion
+func PublisherAccount(st *state.State, snapID string) (snap.StoreAccount, error) {
```

Comment:
PublisherStoreAccount is probably a better name now that is a  public function

Reply:
Renamed it, thanks for the input!